tests/testthat/test-multitables.R

```diff
@@ -277,6 +330,14 @@ with_test_authentication({
         m <- importMultitable(ds2, mult)
         expect_identical(name(m), "Yet another name")
     })
+    
+    test_that("Multitable delete", {
+        mults <- multitables(ds2)
+        delete(mults[["Yet another name"]])
```

Comment:
Should delete require confirmation? Deleting datasets and variables do. 

Can you delete by assigning NULL into the catalog? Looks like you added that method, so we should test it.